docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,357 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d299c1e8",
+   "metadata": {},
+   "source": [
+    "# Finding Label Errors in Object Detection (object detection) Datasets\n",
+    "\n",
+    "This 5-minute quickstart tutorial demonstrates how to find potential label errors in object detection datasets. In these datasets each example contains a bounding box and a class label surrounding a physical object within an image scene. Using this labeled data, we train a model to predict classes of objects in an image and their physical locations. The example notebook used to train the model is avalible here (link).\n",
+    "\n",
+    "Example applications of object detection in computer vision include;\n",
+    "- Medecine, such as identifying foreign objects in x-rays\n",
+    "- Autonomous driving, identifying pedestrians or objects of interest on the road\n",
+    "- Agriculture, picking up crops ready for harvest\n",
+    "\n",
```

Comment:
```suggestion
```

Reply:
Keep tutorial as concise as possible. Anybody using cleanlab to find label errors in object detection dataset already knows what object detection is and has a dataset (aka an application) in mind already.

For blogpost, could be good to list object detection applications for better SEO